portfolio/src/main/webapp/blogpage.css

```diff
@@ -0,0 +1,118 @@
+body{
+    margin: auto;
+}
+
+.landing-section{
+    background-color: cyan;
+    height: 100vh;
+    width: 100vw;
+}
+
+.title{
+    display: flex;
+    justify-content: center;
+    height: 55vh;
+    width: 100vw;
+}
+
+.title h1{
+    font-size: 60pt;
+    font-family: "Montserrat", serif;
+    padding-top: 20vh;
+    padding-bottom: 20vh;
+}
+
+#rect{
+    width: 100vw;
+    height: 45vh;
+    background-color: black;
+}
+
+#pole{ 
+    background-color: red;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    width: 3vw;
+    height: 350vh;
+    margin-left: 49vw;
+}
+
+.blog-section{
+    height: 150vh;
+    width: 100vw;
+    margin-top: -350vh;
+}
+
+.blog-grid{
+    margin-top: 10vh;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.blog{
+    width: 37.5vw;
+    height: 67vh;
+    border: 2px solid blue;
+    padding: 1.5vw 1.5vw 1.5vw;
+    padding-top: 00vw;
```

Comment:
When a value is 0, you can actually use 0 without the units!

Screenshot from the style guide, if you're curious: https://screenshot.googleplex.com/boMcwN8hRVo